### DISCUSSION TITLE
Is it correct to concat the edge_index tensors of graphs batch as GCNConv.forward() input?

### DISCUSSION THREAD

**Discussion by fportera** [pyg-team/pytorch_geometric#8045] [General]
I'm trying to make a GNN to work with several datasets (MUTAG, PROTEINS, etc...), using 2 GCNConv layers for a Neural Network.
While the 'x' input tensor can be batched to a batch-size first dimension, It's not clear to me how to stack/concatenate the relative batch edge_index tensors for batch-size graphs.
If it is possible to use multiple batch graphs with these GCNConv layers....
Thanks for your attention!

**Comment by EdisonLeeeee** [comment]
+ `torch_geometric.data.DataLoader` will concatenate `edge_index` to a batched graph automatically.
+ Yes, you can use `GCNConv`, that makes sense.

**Reply by fportera** [reply]
Many thanks!
